app/addons/fauxton/navigation/stores.js

```diff
@@ -94,8 +107,7 @@ Stores.NavBarStore = FauxtonAPI.Store.extend({
   },
 
   toggleMenu () {
-    app.utils.localStorageSet(FauxtonAPI.constants.LOCAL_STORAGE.SIDEBAR_MINIMIZED,
-                              !this.isMinimized());
+    this._isMinimized = !this._isMinimized;
```

Comment:
We made the decision to open the dashboard with main-nav collapsed, and a clearer trigger for expanding it back out.  I don't think preferences need to be maintained across sessions, especially if the default state is the one providing more screen real estate.

These decisions came out of user feedback regarding confusion on the "hamburger" icon;s functionality, and requests for more visual space to work with.